web/templates/web/demographic-data-update.html

```diff
@@ -119,12 +115,17 @@
         });
         function hideField(field) {
             $(`#id_${field}`).hide().prop('required',false);
-            $(`#id_${field}`).parent().hide(); //hide the parent form group
+            $(`#id_${field}`).parent('.form-group').hide(); // hide the parent form-group div
             $(`label[for=id_${field}]`).hide();
         }
+        function clearField(field) { // hide field and remove any stored data
+            hideField(fieldName);
```

Comment:
Drive-by comment, variables need to be consistent.  I would change "field" argument to fieldName.

Reply:
Darnit thank you! that's what I get for not testing "trivial" changes locally...